Makefile

```diff
@@ -172,7 +172,7 @@ generate-go: $(CONTROLLER_GEN) $(MOCKGEN) $(CONVERSION_GEN) ## Runs Go related g
 generate-manifests: $(CONTROLLER_GEN) ## Generate manifests e.g. CRD, RBAC etc.
 	$(CONTROLLER_GEN) \
 		paths=./api/... \
```

Comment:
we also need `crd:preserveUnknownFields=false` here

Reply:
I'm not sure we do, @vincepri actually removed `crd:preserveUnknownFields=false` in his PR  https://github.com/kubernetes-sigs/cluster-api/pull/2132/files#diff-b67911656ef5d18c4ae36cb6741b7965L263, I think that was required before this change (and that's why it was needed in v1alpha2 and in both the webhook PRs).

Reply:
We don't need the flag for CRDv1, it's assumed to be false. `crd:preserveUnknownFields` needs to be set only for v1beta1 CRDs.